src/main/drivers/inverter.c

```diff
@@ -75,37 +88,37 @@ void enableInverter(USART_TypeDef *USARTx, bool on)
 
 #ifdef INVERTER_PIN_UART1
```

Comment:
Yes, you are right, sorry.

It may be better to pass serialPort_t (or uartPort_t, there is probably little need for inversion for softserial, but there may be a use case) instead of USART_TypeDef. And pass it to initInverter too for consistency